support/fixtures/factories/contxtStack.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const factory = require('rosie').Factory;
+const faker = require('faker');
+
+factory
+  .define('contxtStack')
+  .option('fromServer', false)
+  .attrs({
+    clientId: () => faker.random.uuid(),
```

Comment:
Is this an `auth0` clientId? If so, we often use `faker.internet.password()` for these. (I'm sorry, these are all nitpicky comments)